src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -1101,16 +1137,24 @@ void GenerateFieldDocComment(io::Printer* printer,
   printer->Print(
     " * <code>^def^</code>\n",
     "def", EscapePhpdoc(FirstLineOf(field->DebugString())));
+  if (function_type == kFieldSetter) {
+    printer->Print(" * @param ^php_type^ $var\n",
+      "php_type", PhpTypeName(field, is_descriptor));
```

Comment:
I think that we do want this - the `@return` will be used by IDEs to provide autocompletion, and by documentation generation tools.